### PR TITLE
Change constructor propriety from DriverStandings from Constructor mo…

### PR DESCRIPTION
**Pull request by kletellier**
…del to ArrayCollection of Constructor

### CODE REVIEW

File: src/BrieucThomas/ErgastClient/Model/DriverStanding.php [hunk 1]
```diff
@@ -9,21 +9,23 @@
...
     private $driver;
-    private $constructor;
+    private $constructors;
 
     public function getDriver(): Driver
     {
         return $this->driver;
     }
 
-    public function getConstructor(): Constructor
```
Comment (brieucthomas) [inline src/BrieucThomas/ErgastClient/Model/DriverStanding.php:25]: Can you let this method to keep the backward compatibility ?

Example :

```
public function getConstructor(): ?Constructor
{
   return ($this->constructors instanceof Doctrine\...) ? $this->constructors->first() : null;
}

public function getConstructors(): ?Constructor
{
   // ...
}
```

Thanks!

**Review by brieucthomas — CHANGES_REQUESTED** [review]

**Comment by kletellier** [comment]
Change applied and test updated

File: src/Model/DriverStanding.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?php
```
Comment (brieucthomas) [inline src/Model/DriverStanding.php]: File to remove ?

Comment (kletellier) [reply src/Model/DriverStanding.php]: sorry wrong drag n drop, i push a new version

**Review by brieucthomas — APPROVED** [review]

**Comment by brieucthomas** [comment]
Thanks @kletellier !

**Pull request merged**